ReadMe.md

```diff
@@ -90,9 +90,13 @@ English | Japanese | Kana | Notes
 --------|----------|------|------
 Development | 開発 | かいはつ |
 Developer | 開発者 | かいはつしゃ |
+Programmer | PG/エンジニア | エンジニア | 
+Engineer | PG | ぷろぐらまー |
```

Comment:
On top of your comments, @AlmirKadric, I can add that after discussing with @endowt, "PG" is not a good fit for "Engineer". Because "Programmer" really has 2 Japanese words here, I would prefer 2 distinct entries for them, so the result being:

Programmer | PG | |
Programmer | プログラマー | |
Engineer | エンジニア | |